.github/workflows/lint_python.yml

```diff
@@ -14,7 +14,7 @@ jobs:
                     --max-complexity=10 --max-line-length=255 --show-source --statistics .
       - run: isort --check-only --profile black . || true
       - run: pip install -r test-requirements.txt
-      - run: make test || true
+      - run: tox -e py || make test || true
```

Comment:
you could remove the `make test` it is useless if we only use `tox`. I was thinking of removing the `Makefile` anyway.

Reply:
Done.